.github/workflows/docker-update-images.yaml

```diff
@@ -14,14 +14,16 @@ on:
 jobs:
   update:
     runs-on: ubuntu-latest
+
     steps:
       - uses: actions/checkout@v3
 
       - name: Check and pin updates (if any)
-        run: python3 scripts/update_docker_images.py
+        id: check_update
+        run: python3 scripts/update_docker_images.py >> $"GITHUB_OUTPUT"
 
       - name: Create Pull Request
-        if: ${{ success() }}
+        if: ${{ steps.check_update.outputs.NEEDS_UPDATE == "True" }}
```

Comment:
This `"True"` results in an error actually: 
<img width="1385" alt="image" src="https://user-images.githubusercontent.com/12578616/235260740-b68e9e3d-effe-47a9-baba-1ea79a5203e7.png">


https://github.com/aptos-labs/aptos-core/actions/runs/4824455865